tox.ini

```diff
@@ -80,21 +80,21 @@ deps =
 setenv =
   TEST_JUJU_CHANNEL = 2.9/stable
 changedir = {toxinidir}/tests/functional
-commands = functest-run-suite {posargs:--keep-faulty-model}
```

Comment:
Nitpick: Why do we remove the faulty model? I think it's handy to debug while running the tests manually via `make functional`.

Reply:
There is no need for keep-faulty-model on CI.

Also keep-faulty-model seems not clean the model probably.

See [here](https://github.com/canonical/charm-prometheus-juju-exporter/actions/runs/8534519674/job/23379126216)